admin/lib/solidus_admin/testing_support/feature_helpers.rb

```diff
@@ -22,6 +22,10 @@ def find_row(text)
         find('table tbody tr', text:)
       end
 
+      def click_row(text)
```

Comment:
You are actually clicking a cell ;) Can we either change the selector or adjust the name?

Reply:
I just figured we are not using a link in the cell, but rely on the click handler on the row. Which is bad UX and not semantic at all actually. But this is a whole different story we see all over the new admin and should be rethought.